README.md

```diff
@@ -100,7 +105,29 @@ Each lion should have...
 Create a new lion instance with the name "Simba".
 
 ```ruby
-# code here
+class Animal
+  attr_accessor :name
+  def initialize(name)
+    @name = name
+  end
+  def greet
+    puts "Hello my name is #{name}"
+  end
+
+end
+
+
+class Lion
+  def initialize(name, pack)
+    super(name)
+    @pack = pack
+  end
+
+
+end
+
+pumba = Animal.new("Pumba")
+simba = Lion.new("Simba")
 ```
```

Comment:
To clean up the code, you want your Lion class to inherit from Animal Class. 
Answer:
class Lion < Animal
  attr_accessor :king
  @@pack = []

  def initialize name
    super(name)
    if name == "Simba"
      @king = true
    else
      @king = false
    end
    @@pack << self
  end
end